src/main/java/bossmonster/BossMonster.java

```diff
@@ -0,0 +1,74 @@
+package bossmonster;
+
+import bossmonster.VO.HP;
+
+public class BossMonster {
+
+    private HP hp;
+    private String avatar;
+    private static final String NORMAL_FORM
+                = "   ^-^\n"
+                + " / 0 0 \\\n"
+                + "(   \"   )\n"
+                + " \\  -  /\n"
+                + "  - ^ -";
+    private static final String DAMAGED_FORM
+                = "   ^-^\n"
+                + " / x x \\\n"
+                + "(   \"\\  )\n"
+                + " \\  ^  /\n"
+                + "  - ^ -";
+    private static final String VICTORY_FORM
+                = "   ^-^\n"
+                + " / ^ ^ \\\n"
+                + "(   \"   )\n"
+                + " \\  3  /\n"
+                + "  - ^ -";
```

Comment:
static 필드가 위로 올라가면 더 보기 좋을거 같습니다 👍

```suggestion
    private static final String NORMAL_FORM
                = "   ^-^\n"
                + " / 0 0 \\\n"
                + "(   \"   )\n"
                + " \\  -  /\n"
                + "  - ^ -";
    private static final String DAMAGED_FORM
                = "   ^-^\n"
                + " / x x \\\n"
                + "(   \"\\  )\n"
                + " \\  ^  /\n"
                + "  - ^ -";
    private static final String VICTORY_FORM
                = "   ^-^\n"
                + " / ^ ^ \\\n"
                + "(   \"   )\n"
                + " \\  3  /\n"
                + "  - ^ -";
                
    private HP hp;
    private String avatar;
```

Reply:
😓코드 컨벤션에 맞게 수정한 직후에 실수를 해버렸네요..
코드 컨벤션을 최대한 외워서 언제든 컨벤션을 지키도록 노력해야겠습니다.🥲